load_webp.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021 Tobias Stoeckmann <tobias@stoeckmann.org>
```

Comment:
The copyright is added to each file to clarify who wrote the major parts of it. In this case, it would be your copyright, not mine.